setup/steps/apply_monitoring_user_password.go

```diff
@@ -0,0 +1,86 @@
+package steps
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	survey "github.com/AlecAivazis/survey/v2"
+	"github.com/lib/pq"
+	"github.com/pganalyze/collector/config"
+	s "github.com/pganalyze/collector/setup/state"
+	mainUtil "github.com/pganalyze/collector/util"
+)
+
+var ApplyMonitoringUserPasswd = &s.Step{
+	Description: "Apply monitoring user password",
+	Check: func(state *s.SetupState) (bool, error) {
+		cfg, err := config.Read(
```

Comment:
It would be helpful to explain why where are using the main collector code for reading the config here (i.e. to be able to use `GetPqOpenString` below